django_evaluation/settings/local.py

```diff
@@ -241,17 +241,11 @@ def _set_favicon(html_color: str, project_root: Path) -> None:
 SERVER_EMAIL = os.environ.get("SERVER_EMAIL", "freva@dkrz.de")
 DEFAULT_FROM_EMAIL = SERVER_EMAIL
 
-EMAIL_HOST = os.environ.get("EMAIL_HOST", "mailhost.dkrz.de")
-
-EMAIL_PORT = int(os.environ.get("EMAIL_PORT", "25"))
-if EMAIL_PORT == 25:
-    EMAIL_HOST_USER = ""
-    EMAIL_HOST_PASSWORD = ""
-else:
-    email_secrets = _read_secret()
-    EMAIL_HOST_USER = email_secrets.get("username")
-    EMAIL_HOST_PASSWORD = email_secrets.get("password")
-
+EMAIL_HOST = "smtp.sendgrid.net"
+EMAIL_PORT = "587"
+EMAIL_USE_TLS = True
+EMAIL_HOST_USER = "apikey"
+EMAIL_HOST_PASSWORD = os.getenv("EMAIL_HOST_PASSWORD")
```

Comment:
what do u think to keep `_read_secret()` and define it as:

```python
    EMAIL_HOST_USER = email_secrets.get("username") or os.getenv("EMAIL_HOST_USER")
    EMAIL_HOST_PASSWORD = email_secrets.get("password") or os.getenv("EMAIL_HOST_PASSWORD")
```

Reply:
All instances should be using the same sendgrid thingy. If we hit the 100 emails/month limit we can reevaluate. But I pretty much doubt that this will ever happen.